swagger-ui/swagger-ui.js

```diff
@@ -42,6 +42,18 @@ this["Handlebars"]["templates"]["content_type"] = Handlebars.template({"1":funct
 },"useData":true});
 'use strict';
 
+var replaceResponse = function(a, b, element) {
+  if (!element) element = document.body;
+  var nodes = element.childNodes;
+  for (var n=0; n<(nodes.length ? nodes.length : 0); n++) {
+    if (nodes[n].nodeType == Node.TEXT_NODE) {
+      var reg = new RegExp(a, 'gi');
+      nodes[n].textContent = nodes[n].textContent.replace(reg, b);
+    } else {
+      replaceResponse(a, b, nodes[n]);
```

Comment:
hmm - could we use jquery for finding the node that matches the criteria (e.g. https://api.jquery.com/contains-selector/)? Just wondering whether there's a more performant approach than recursive decent on the DOM.